src/utils/translation.ts

```diff
@@ -41,6 +43,9 @@ export async function init() {
     .use(intervalPlural)
     .init({
       resources: {
+        ta:{
```

Comment:
Please place `ta` in the same sequence like in the imports

Reply:
ok I Will Change That

Reply:
I have done that